utils/vscode/esbuild.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+ * Exceptions. See /LICENSE for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+/**
+ * This supports using esbuild to bundle the extension for releases.
```

Comment:
This comment wasn't enough for me to figure out what this file is doing.  Can you add some more description here? Specifically, it looks like this is a program that can be run using `node`; what is its command-line syntax and what does it do when run?

It would also have helped me for this file to point at the invocation of this from the scripts in `package.json`, and maybe to also point at the uses of the scripts in `.vscode/tasks.json`.

Reply:
Linked back to https://code.visualstudio.com/api/working-with-extensions/bundling-extension

I'm not sure I'm getting what you're looking for exactly though, maybe you can suggest changes?

Reply:
To be clear, I'm mostly re-using code here. I didn't write this myself, so I don't have a strong grasp of it beyond that it fixes a warning from vsce.

Reply:
@chandlerc Updated file per discussion.